encord/ontology.py

```diff
@@ -0,0 +1,138 @@
+        """
+        Get the title of the ontology.
+        """
+        ontology_instance = self._get_ontology_instance()
+        return ontology_instance.title
+
+    @title.setter
+    def title(self, value: str) -> None:
+        ontology_instance = self._get_ontology_instance()
+        ontology_instance.title = value
+
+    @property
+    def description(self) -> str:
+        """
+        Get the description of the ontology.
+        """
+        ontology_instance = self._get_ontology_instance()
+        return ontology_instance.description
+
+    @description.setter
+    def description(self, value: str) -> None:
+        ontology_instance = self._get_ontology_instance()
+        ontology_instance.description = value
+
+    @property
+    def created_at(self) -> datetime.datetime:
+        """
+        Get the time the ontology was created at.
+        """
+        ontology_instance = self._get_ontology_instance()
+        return ontology_instance.created_at
+
+    @property
+    def last_edited_at(self) -> datetime.datetime:
+        """
+        Get the time the ontology was last edited at.
+        """
+        ontology_instance = self._get_ontology_instance()
+        return ontology_instance.last_edited_at
+
+    @property
+    def structure(self) -> OntologyStructure:
+        """
+        Get the structure of the ontology.
+        """
+        ontology_instance = self._get_ontology_instance()
+        return ontology_instance.structure
+
+    def refetch_data(self) -> None:
+        """
+        The Ontology class will only fetch its properties once. Use this function if you suspect the state of those
+        properties to be dirty.
+        """
+        self._ontology_instance = self._get_ontology()
+
+    def save(self) -> None:
+        """
+        Sync local state to the server, if updates are made to structure, title or description fields
+        """
+        if self._ontology_instance:
+            payload = dict(**self._ontology_instance)
+            payload["editor"] = self._ontology_instance.structure.to_dict()  # we're using internal/legacy name here
+            payload.pop("structure", None)
+            self._querier.basic_put(OrmOntology, self._config.resource_id, payload)
+
+    def add_users(self, user_emails: List[str], user_role: OntologyUserRole) -> List:
+        """
+        Add users to ontology
+
+        Args:
+            user_emails: list of user emails to be added
+            user_role: the user role to assign to all users
+
+        Returns:
+            OntologyUser
+
+        Raises:
+            AuthorisationError: If the ontology API key is invalid.
+            ResourceNotFoundError: If no ontology exists by the specified ontology EntityId.
+            UnknownError: If an error occurs while adding the users to the ontology
+        """
+        return self._client.add_users(user_emails, user_role)
+
+    def copy_ontology(self, new_title: str, new_description: str) -> str:
```

Comment:
you're not passing the arguments

Reply:
see above: at some point i decided to postpone this to a follow-up task, but forgot to remove the start that I've made